vendor/modules.txt

```diff
@@ -56,6 +56,9 @@ github.com/go-openapi/jsonreference/internal
 # github.com/go-openapi/swag v0.22.3
 ## explicit; go 1.18
 github.com/go-openapi/swag
+# github.com/go-test/deep v1.1.0
+## explicit; go 1.16
+github.com/go-test/deep
```

Comment:
issue: we use `go-cmp` here already, we can use it or `testify/assert`instead of pulling another dependency